src/Microsoft.Azure.Batch.SoftwareEntitlement.Client.Native/SoftwareEntitlementClient.cpp

```diff
@@ -656,8 +595,97 @@ class Curl
 
         throw Exception(GetErrorMessage(code));
     }
+
+    static std::unique_ptr<Entitlement> GetEntitlement(
+        std::string url,
```

Comment:
`url` can also be passed by const reference

Reply:
done